singlestoredb_cloud/manifest.json

```diff
@@ -0,0 +1,55 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "c7638089-0864-4ddc-bd32-b731c58fe567",
+  "app_id": "singlestoredb-cloud",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send your SinglestoreDB Cloud metrics to DataDog.",
+    "title": "SingleStoreDB Cloud",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "SinglestoreDB Cloud - Dashboard",
+        "image_url": "images/singlestoredb-cloud-dashboard.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Data Store",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Offering::Integration",
+      "Submitted Data Type::Metrics"
```

Comment:
Below you've defined a log source. Is this integration sending in logs?

Reply:
We were initially planning, but scoped it down for V1 of the integration; I have update the manifest to exclude logs